test-framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -161,6 +161,25 @@ public static void restoreContentType() {
         Requests.INDEX_CONTENT_TYPE = XContentType.JSON;
     }
 
+    private static boolean TESTS_SEED_PROPERTY_SET = false;
+    @BeforeClass
+    @SuppressForbidden(reason = "sets tests.seed")
```

Comment:
The method `System#getProperty(String, String)` is [forbidden](https://github.com/elastic/elasticsearch/blob/914486795f0c97cc0167141d15a4f10cd3696089/buildSrc/src/main/resources/forbidden/all-signatures.txt#L111). The [whitelisting](https://github.com/elastic/elasticsearch/pull/15299/files#diff-f4697efc979568b5987de63fbc784ab9R68) is merely so that when we do circumvent forbidden APIs on `System#getProperty(String, String)`, the security policy will allow us to write to the key `tests.seed`.